09-mapping.Rmd

```diff
@@ -687,11 +720,14 @@ library(globe)
 ```
 
 **tmap** provides a powerful interface for creating a wide range of static maps (section \@ref(interactive-maps) shows how **tmap** also supports interactive maps).
+<!-- if this section is on static map production, it should be part of section static maps -->
 But there are many other options for creating static maps.
 The aim of this section is to provide a taster of some of these and pointers for additional resources: map making is a surprisingly active area of R package development so there is more to learn than can be covered here.
 
+<!-- maybe this should be separated from basic-map and basic-map-raster
+Also really doesn't belong here since the section header is other mapping packages-->
 The most mature option is to use `plot()` methods provided by core spatial packages **sf** and **raster**, covered in sections \@ref(basic-map) and \@ref(basic-map-raster) respectively.
```

Comment:
Maybe this should be separated from basic-map and basic-map-raster.
This doesn't belong here since the section header is "other mapping packages".